### DISCUSSION TITLE
Does Jolt.Plane2D only affect Motion , Not affect Collision & Contact  ?

### DISCUSSION THREAD

**Discussion by finscn** [jrouwe/JoltPhysics#713] [General]
I search the keyworld Plane2D & mAllowedDOFs , I found they only  work in the codes about Motion.

Does it mean "Check Collision" & "Solve (Contact) Constraints" are still in 3D ? 
So the performace is not as good as  box2D ?

**Comment by jrouwe** [comment]
Yes, collision detection is in 3D.

I haven't compared Box2D vs Jolt.
